lib/system.nim

```diff
@@ -691,7 +691,9 @@ proc len*(x: string): int {.magic: "LengthStr", noSideEffect.}
 
 proc len*(x: cstring): int {.magic: "LengthStr", noSideEffect.}
   ## Returns the length of a compatible string. This is sometimes
-  ## an O(n) operation.
+  ## an O(n) operation. On the JS backend this may count Unicode
+  ## code points instead of characters, similar to `unicode.runeLen
+  ## <unicode.html#runeLen%2Cstring>`_
```

Comment:
thanks!
if you don't mind also changing these (IIRC you introduced the last 2 in your recent PR)

```
doc/backends.rst:85:2:<https://nim-lang.org/docs/lib.html#pure-libraries-modules-for-js-backend>`_
lib/pure/cstrutils.nim:72:7:  ## <https://nim-lang.org/docs/strutils.html#cmpIgnoreStyle%2Cstring%2Cstring>`_ instead.
lib/pure/cstrutils.nim:94:7:  ## <https://nim-lang.org/docs/strutils.html#cmpIgnoreCase%2Cstring%2Cstring>`_ instead.
```

Reply:
I did